run-test.py

```diff
@@ -722,9 +722,8 @@ def kill(pod, proc_name):
 
 def restore_heavy_from_backup(heavy_pod):
     info("Restoring heavy from backup at pod#..."+str(heavy_pod))
-    kill(heavy_pod, "backupmanager")
     ssh(heavy_pod, "cd "+INSPATH+" && " +
-        "./bin/backupmanager prepare_backup -d ./heavy_backup/ && " +
+        "rsync -aW --no-compress data heavy_backup" +
         "rm -r data && cp -r heavy_backup data")
```

Comment:
I believe heavy_backup is already created - see heavy config, postprocessbackupcmd. Thus you just need rm -r + cp -r.